db/seeds.rb

```diff
@@ -5,11 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-user = User.first
-urls = ['csnipp.com', 'redpanthers.co', 'google.com']
 
-urls.each do |url|
-  user.websites.create!(url: url)
+user = User.first_or_create!(name: 'abc', email: 'abc@mail.com', password: '123456',                    password_confirmation: '123456')
+urls = ['abc.com']
+names = ['list']
+
+names.each do |name|
+  collection = user.collections.create!(name: name)
+  urls.each do |url|
+    collection.websites.create!(url: url)
+  end
 end
```

Comment:
In seeds.rb, you don't really need all this complexity, since you know what data you want to add. You can write it like this:

```ruby
user = User.create!(name: 'abc', email: 'abc@mail.com', password: '123456', password_confirmation: '123456')
collection = user.collections.create!(name: 'First Collection')
collection.websites.create!(url: 'https://redpanthers.co')
```

When adding seed data, it would be good to add valid url, since you will be using it in development.